data/tutorials/language/1ds_01_arrays.md

```diff
@@ -0,0 +1,158 @@
+---
+id: arrays
+title: Arrays
+description: >
+  The standard library's Array module
+category: "Data Structures"
+---
+
+# Arrays
+
+## Introduction
+
+In OCaml, arrays are collections of elements of the same type that are stored in contiguous memory locations. They are mutable, each value can be updated, but they have a fixed size; which set them apart from lists.
+
+Despite these differences many of the functions readily available on arrays are similar to the ones available for lists, please refer to the [List tutorial](https://ocaml.org/docs/lists) and documentation for more details about these functions.
```

Comment:
I think it'd be helpful to mention some example before urging new users/readers back to the List tutorial.

> Despite these differences, many of the functions readily available on arrays are similar to the ones available for lists such as X, Y, and Z. Please refer to the [List tutorial](https://ocaml.org/docs/lists) and documentation for more details about these functions.